pages/dr10/files.rst

```diff
@@ -0,0 +1,1066 @@
+.. title: Legacy Survey Files
+.. slug: files
+.. tags: 
+.. has_math: yes
+
+.. |sigma|    unicode:: U+003C3 .. GREEK SMALL LETTER SIGMA
+.. |sup2|     unicode:: U+000B2 .. SUPERSCRIPT TWO
+.. |chi|      unicode:: U+003C7 .. GREEK SMALL LETTER CHI
+.. |delta|    unicode:: U+003B4 .. GREEK SMALL LETTER DELTA
+.. |deg|    unicode:: U+000B0 .. DEGREE SIGN
+.. |times|  unicode:: U+000D7 .. MULTIPLICATION SIGN
+.. |plusmn| unicode:: U+000B1 .. PLUS-MINUS SIGN
+.. |Prime|    unicode:: U+02033 .. DOUBLE PRIME
+.. |gtapprox| unicode:: U+02273 .. GREATER-THAN OR EQUIVALENT TO
+
+.. class:: pull-right well
+
+.. contents::
+
+Directory Structures
+====================
+
+For Web Access
+--------------
+
+| **Top level directory:**
+| https://portal.nersc.gov/cfs/cosmo/data/legacysurvey/dr10/south/
+| **Top level directory for sweep catalogs:**
+| https://portal.nersc.gov/cfs/cosmo/data/legacysurvey/dr10/south/sweep/
+
+
+At NERSC (for collaborators)
+----------------------------
+
+| **Top level directory:**
+| /global/cfs/cdirs/cosmo/data/legacysurvey/dr10/south/
```

Comment:
Again, this is not the actual top-level directory.

Reply:
Again, thanks for noticing this oversight. I changed this to `/global/cfs/cdirs/cosmo/data/legacysurvey/dr10/`.